src/controllers/user.controller.js

```diff
@@ -3,37 +3,71 @@ import { BcryptUtility } from '../utils/bcrypt.util.js';
 import { JwtUtility } from '../utils/jwt.util.js';
 
 export class UserController {
-	static async registerUser(req, res) {
-		try {
-			const user = { ...req.body };
-			user.password = await BcryptUtility.hashPassword(req.body.password);
-			const { id, email } = await UserService.register(user);
-			const userData = { id, email };
-			const userToken = JwtUtility.generateToken(userData);
-			return res.status(201).json({ user: userData, token: userToken });
-		} catch (err) {
-			return res.status(500).json({
-				error: err.message,
-				message: 'Failed to register a new user',
-			});
-		}
-	}
+  static async registerUser(req, res) {
+    try {
+      const user = { ...req.body };
+      user.password = await BcryptUtility.hashPassword(req.body.password);
+      const { id, email } = await UserService.register(user);
+      const userData = { id, email };
+      const userToken = JwtUtility.generateToken(userData);
+      return res.status(201).json({ user: userData, token: userToken });
+    } catch (err) {
+      return res.status(500).json({
+        error: err.message,
+        message: 'Failed to register a new user',
+      });
+    }
+  }
 
-	static async loginUser(req, res) {
-		try {
-			const {id, email, firstname, lastname, gender, role, status, avatar, verified} = req.user;
+  static async updatePassword(req, res) {
+    try {
+      const password = await BcryptUtility.hashPassword(req.body.new_password);
+      await UserService.updatePassword(password, req.user.id);
+      return res.status(201).json({ success: 'Updated successfully' });
```

Comment:
Correct the response
- Status code should success, because you're not creating
- Change the key of your response
- Update your message